doc/article/en-US/binding-basics.md

```diff
@@ -25,10 +25,10 @@ Aurelia supports binding HTML and SVG attributes to JavaScript expressions. Attr
 * `attribute`:  an HTML or SVG attribute name.
 * `command`: one of Aurelia's attribute binding commands:
   * `one-time`: flows data one direction: from the view-model to the view, **once**.
-  * `one-way` / `to-view`: flows data one direction: from the view-model to the view.
+  * `to-view` / `one-way`: flows data one direction: from the view-model to the view.
```

Comment:
perhaps it should me made clear, that
> to-view should definitely be the preferred command

(as @jdanyow said) - in the other words, `one-way` is deprecated (but can still be used for backwards compatibility reasons), but `to-view` is the preferred command?